sh/tests/basic_functionality.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+echo "Test 1: Basic functionality"
+
+ORIGINAL_DIR=$(cd "$(dirname "$0")/.." && pwd)
+
+TEST_DIR=$(mktemp -d)
+cd "$TEST_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)